### PR TITLE
Bah 8940 change order

### PR DESCRIPTION
**Pull request by aaronjlech**
## Description
This PR covers:
- removes Placeholder individual unemployability question page
- places 8940/4192 inline with correct individual unemployability page
- consolidates all of 8940 into a single file outside of all-claims config
- Adds production flag/conditional to remove 8940 when all-claims goes live on production.
- Removes Unemployability from content page if the user uploads or answers 8940/4192 questions.

## Testing done
- [x] local testing

## Screenshots
image
image
image

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

see story: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15469

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx [hunk 1]
```diff
@@ -89,6 +89,9 @@ const individualUnemployabilityContent = (
 );
 
 export default function summaryDescription({ formData }) {
+  const showUnemployableContent =
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx]: I'm not sure this is the best way to get this to still show up/not show up depending on the outcome 🤔

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 2]
```diff
@@ -76,9 +76,6 @@ import {
   vaEmployee,
   summaryOfEvidence,
   fullyDevelopedClaim,
-  unemployabilityStatus,
-  unemployabilityFormIntro,
-  workBehaviorChanges,
```
Comment (imeecuison) [inline src/applications/disability-benefits/all-claims/config/form.js]: I think workBehaviorChanges needs to stay in all-claims/config/form.js

File: src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx (hunk 1 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx]:
```
const showUnemployableContent =
    formData['view:unemployable'] &&
    formData['view:unemployabilityUploadChoice'] === 'answerQuestions';
```
Should it be like this for individualUnemployabilityContent to show up?

**Review by imeecuison — CHANGES_REQUESTED** [review]

File: src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx (hunk 1 above)
Comment (aaronjlech) [reply src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx]: Well the goal is to get it not show up if they run through our 8940/4192 form

File: src/applications/disability-benefits/all-claims/config/form.js (hunk 2 above)
Comment (aaronjlech) [reply src/applications/disability-benefits/all-claims/config/form.js]: Great catch!!!

File: src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx (hunk 1 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx]: Oh! Got it!

**Review by imeecuison — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]
looks good- just one question.

**Comment by aaronjlech** [comment]
@rianfowler Not seeing your question?

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 3]
```diff
@@ -606,6 +606,13 @@ export const showPtsdAssaultConclusion = form =>
   _.get('view:selectablePtsdTypes.view:mstPtsdType', form, false) ||
   _.get('view:selectablePtsdTypes.view:assaultPtsdType', form, false);
 
+export const needsToEnterUnemployability = formData =>
```
Comment (rianfowler) [inline src/applications/disability-benefits/all-claims/utils.jsx]: do these need unit tests?

Comment (aaronjlech) [reply src/applications/disability-benefits/all-claims/utils.jsx]: Good Call. Tests added.

**Pull request merged**